webpack.config.scripts.js

```diff
@@ -1,14 +1,20 @@
 const path = require("path");
-
-const name = "permissions-display";
+const process = require("process");
```

Comment:
`process` is a global value in nodejs context, you don't need to require it

```suggestion
```

Reply:
Raised f/u:

https://github.com/SkynetLabs/skynet-mysky/issues/308

Thank you!